airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx

```diff
@@ -127,33 +114,40 @@ const TransformationForm: React.FC<TransformationProps> = ({
                   id: "form.repositoryUrl.placeholder",
                 })}
               />
-            </Label>
-          </Column>
+            </ControlLabels>
+          </div>
 
-          <Column>
-            <Label label={<FormattedMessage id="form.transformationType" />}>
+          <div className={styles.column}>
+            <ControlLabels className={styles.label} label={<FormattedMessage id="form.transformationType" />}>
```

Comment:
What does this get us? Just curious why the change.

Reply:
Our team decided to do not use styled-components at all and refactor existing

Reply:
Oh, Label was just a styled component around ControlLabels. Got it!